### PR TITLE
linux-testing: 5.13-rc6 -> 5.14-rc3

### PR DESCRIPTION
**Pull request by ymatsiuk**
###### Motivation for this change
version bump

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Relase notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by ymatsiuk** [comment]
Once https://github.com/anthraxx/linux-hardened/tree/5.13 is released, [patches.json](https://github.com/NixOS/nixpkgs/blob/47787b24f425579151a09869364c5fa827ed80bf/pkgs/os-specific/linux/kernel/hardened/patches.json#L20) must be updated and this PR is ready to go.

File: nixos/tests/kernel-generic.nix [hunk 1]
```diff
@@ -6,23 +6,25 @@
...
+        {
+          boot.kernelPackages = linuxPackages;
+        };
+
+      testScript =
+        ''
+          assert "Linux" in machine.succeed("uname -s")
+          assert "${linuxPackages.kernel.modDirVersion}" in machine.succeed("uname -a")
+        '';
+    })
```
Comment (Ma27) [inline nixos/tests/kernel-generic.nix:26]: If you think that thsi formatting is really needed, please do it in a separate commit.

**Review by Ma27 — CHANGES_REQUESTED** [review]
This doesn't seem to evaluate.

**Comment by TredwellGit** [comment]
We should refuse to update hardened to 5.13 unless it is not dependent on linux_latest. It has been discussed many times how the hardened patches make everything less secure because of how long they take to fix known security vulnerabilities in order to provide mitigations for unknown security vulnerabilities.

**Comment by ymatsiuk** [comment]
Seems like I've committed to something I'm not be able to complete. I dropped the commit about `5.12` removal. This is only version bump for `linux_testing` now 🙂
